presto-accumulo/src/test/java/io/prestosql/plugin/accumulo/TestAccumuloIntegrationSmokeTest.java

```diff
@@ -16,15 +16,19 @@
 import com.google.common.collect.ImmutableMap;
 import io.prestosql.testing.AbstractTestIntegrationSmokeTest;
 import io.prestosql.testing.MaterializedResult;
+import io.prestosql.testing.QueryRunner;
 
+import static io.prestosql.plugin.accumulo.AccumuloQueryRunner.createAccumuloQueryRunner;
```

Comment:
nit: This static import should be separated commit or written in commit description. How about creating a commit to import `createXXXQueryRunner` method as static including other classes?

Reply:
i was changing this piece of code anyway, hence imported.